app/workers/metrics/record_daily_usage_measurables_worker.rb

```diff
@@ -0,0 +1,17 @@
+module Metrics
+  class RecordDailyUsageMeasurablesWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform
+      articles_min_15_score_past_24h = Article.published.where("score >= 15").size
+      DataDogStatsClient.count("measurables.articles_min_15_score_past_24h", articles_min_15_score_past_24h, tags: { resource: "articles" })
```

Comment:
`measurables` seems a bit vague and since everything we send to Datadog we are measuring what do you think about labels such as `articles.min_15_score_past_24h`, `articles.first_past_24h`, `users.new_min_1_comment_past_24h` to be more descriptive. Btw I like the resource tag.

Reply:
That works for me. I really wasn't sure what that value was supposed to mean with regards to the dot.

Reply:
That's the convention for "namespacing" stuff in data dog which is why I use it. Usually the first word is the highlest level then gets more descriptive from there.